datasets/so_stacksample/so_stacksample.py

```diff
@@ -0,0 +1,173 @@
+            version=VERSION,
+            description="This part of the dataset contains only tags of the questions in the question part of the StackSample dataset.",
+        ),
+    ]
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+        You must have a kaggle account. Go to https://www.kaggle.com/stackoverflow/stacksample
+        and manually download the language of your interest. Once it is downloaded,
+        go to the place where you downloaded it and unzip the folder. Three files named
+        `Answers.csv`, `Questions.csv`, and `Tags.csv` will have appeared in your Downloads folder
+        or whichever folder your browser chooses to save files to.
+        so_stacksample can then be loaded using the following command
+        `datasets.load_dataset("so_stacksample", "<csv_file_name>",data_dir="<path/to/folder>")`,
+        where `<path/to/folder> is the path to the unzipped folder. Example if you downloaded
+        and unzipped the folder in your downloads folder:
+        `datasets.load_dataset("so_stacksample", "Answers", data_dir="/home/<user>/Downloads")`
+        will load the `Answers.csv` dataset.
+        """
+
+    def _info(self):
+        if self.config.name == "Answers":  # This is the name of the configuration selected in BUILDER_CONFIGS above
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "OwnerUserId": datasets.Value("int32"),
+                    "CreationDate": datasets.Value("string"),
+                    "ParentId": datasets.Value("int32"),
+                    "Score": datasets.Value("int32"),
+                    "Body": datasets.Value("string"),
+                }
+            )
+        elif self.config.name == "Questions":
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "OwnerUserId": datasets.Value("int32"),
+                    "CreationDate": datasets.Value("string"),
+                    "ClosedDate": datasets.Value("string"),
+                    "Score": datasets.Value("int32"),
+                    "Title": datasets.Value("string"),
+                    "Body": datasets.Value("string"),
+                }
+            )
+        else:
+            features = datasets.Features(
+                {
+                    "Id": datasets.Value("int32"),
+                    "Tag": datasets.Value("string"),
+                }
+            )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        path_to_manual_file = os.path.join(
+            os.path.abspath(os.path.expanduser(dl_manager.manual_dir)), self.config.name + ".csv"
+        )
```

Comment:
can you raise an error here if the file doesn't exist ?

You can find an example here : https://github.com/huggingface/datasets/blob/master/datasets/arxiv_dataset/arxiv_dataset.py#L104-L109